pkg/local_object_storage/pilorama/interface.go

```diff
@@ -17,7 +17,8 @@ type Forest interface {
 	// Internal nodes in path should have exactly one attribute, otherwise a new node is created.
 	TreeAddByPath(d CIDDescriptor, treeID string, attr string, path []string, meta []KeyValue) ([]LogMove, error)
 	// TreeApply applies replicated operation from another node.
-	TreeApply(d CIDDescriptor, treeID string, m *Move) error
+	// If background is true, TreeApply will first check whether an operation exists.
+	TreeApply(d CIDDescriptor, treeID string, m *Move, background bool) error
```

Comment:
are there any pros why `background` is better than `sync`?

Reply:
`sync` is too generic, renamed to `backgroundSync`